### PR TITLE
fix(token-grant): apply API-Key grant in router navigation guard & create getAccessToken method in SpaceConnector

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/router/index.ts [hunk 1]
```diff
@@ -68,6 +68,26 @@ export class SpaceRouter {
...
+            if (accessToken && isTokenAlive && !isDuplicatedRoute) {
+                let scope: string;
+                if (to.name?.startsWith('admin.')) {
+                    scope = 'DOMAIN';
+                } else if (to.params.workspaceId) {
+                    scope = 'WORKSPACE';
+                } else scope = 'USER';
+                const grantRequest = {
+                    scope,
+                    workspace_id: to.params.workspaceId,
```
Comment (sulmoJ) [inline apps/web/src/router/index.ts:84]: Question: Are there any cases where params do not exist at all? If you have any, you may need a defense code.

File: apps/web/src/store/modules/user/actions.ts [hunk 2]
```diff
@@ -152,13 +151,12 @@ export const signOut = (): void => {
...
+export const grantRole: Action<UserState, any> = async ({ commit }, grantRequest: Omit<TokenGrantParameters, 'grant_type'>) => {
     const response = await SpaceConnector.clientV2.identity.token.grant<TokenGrantParameters>({
         grant_type: 'API_KEY',
-        scope: grantScope,
-        token: accessToekn,
-        workspace_id,
+        scope: grantRequest.scope,
+        token: grantRequest.token,
+        workspace_id: grantRequest.workspace_id,
     }, { skipAuthRefresh: true });
```
Comment (sulmoJ) [inline apps/web/src/store/modules/user/actions.ts:160]: Issue: I think we need try-catch code in this case.

**Review by sulmoJ — COMMENTED** [review]
I left little comments!

File: apps/web/src/router/index.ts (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/router/index.ts:84]: Empty `params` alway exists. And I handle no-workspaceId case above scope assignment!
In no-workspaceId case, scope is `USER`!

**Pull request merged**